content/s12_sending_emails_and_email_verification/s12_p01_email_support_for_webapps.md

```diff
@@ -19,4 +19,4 @@ All you need for email support is two main things:
 As for 1., you could host one yourself, but for development, it's much easier to use an external server. In the next lesson, you'll be guided through the steps to setup a Google Gmail account so you can start playing around with emails right away.
 The settings you'll learn about in the lesson after that, but let's get you set up with a secondary email account!
 
-![](https://images.unsplash.com/photo-1534803522048-835d05301b08?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=2018&q=80)
\ No newline at end of file
+![](https://images.unsplash.com/photo-1534803522048-835d05301b08?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=2018&q=80)
```

Comment:
I like both of these.

Reply:
> 
> 
> looks like some of this is from an earlier commit so I can't comment in-line with it, but I'm gonna add comments with links to the diff:

Shoot, I thought it would work! You can "expand" the files either?